cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/documentstore/DatabaseImplExceptionsTest.java

```diff
@@ -166,7 +166,7 @@ public void deleteDocumentFromRevisionShouldThrowDocumentStoreException() throws
         this.datastore.deleteDocumentFromRevision(dr);
     }
 
-    // updateDocumentFromRevision after we remove the underlying SQL database should throw
+    // deleteDocumentFromRevision after we remove the underlying SQL database should throw
```

Comment:
This one's still wrong, it should be `deleteDocument`, not `deleteDocumentFromRevision`